net/udpspeeder/files/udpspeeder-init

```diff
@@ -79,6 +80,11 @@ start_instance() {
 		procd_append_param command --disable-obscure
 	fi
 
+	if [ ! -z "${key}" ];
```

Comment:
You can simplify with

```
[ -z "${key}"  || procd_append_param command --key "${key}"
```

Reply:
Done. It is looks inconsistent with other condition branch in this file.